src/fontra/views/editor/panel-selection-info.js

```diff
@@ -177,10 +177,11 @@ export default class SelectionInfoPanel extends Panel {
               : "/tabler-icons/lock-open-2.svg",
           "onclick": (event) => this._toggleGlyphLock(varGlyphController.glyph),
           "data-tooltip": this.fontController.readOnly
-            ? "Glyph is read-only"
+            ? translate("sidebar.selection-info.glyph-locking.tooltip.read-only")
```

Comment:
Same, pls put the switch(es) inside the translate() call.

Reply:
done.